### DISCUSSION TITLE
Supervisor default production configuration

### DISCUSSION THREAD

**Discussion by mriegler** [pimcore/pimcore#17337] [Q&A / Support] [answered]
While the skeleton has an example supervisor configuration, it's explicitly marked as not for production. Yet, I also cannot find any example for the correct configuration in the documentation.

Obviously, everyone may need to customize this configuration over time, but it would be immensely helpful to have an example of a production supervisor configuration as a jumping off point. Most people aren't running a hyper customized, globally scaled, 1M user site, after all.

**Comment by dpfaffenbauer** [comment, marked as answer]
We run that in multiple different production environments without any issue. We sometimes have more workers per queue and tweak the timeouts a bit. But mostly it is the same supervisor config
